src/components/CreateAccount/index.js

```diff
@@ -1,115 +1,314 @@
+                  pattern: /^(?=.*[0-9]+.*)(?=.*[a-zäöüß]+.*)(?=.*[A-ZÄÖÜ]+.*)[\w\W]{10,}$/,
+                })}
+                type="password"
+                ariaInvalid={!!errors.password}
+                inputClassName="create-account-input-text"
+                labelClassName={
+                  errors.password
+                    ? 'create-account-errors-field'
+                    : 'create-account-input-field'
+                }
+              />
+              <div className="info-circle">
+                <button
+                  type="button"
+                  onClick={openPasswordModal}
+                  className="info-circle-button"
+                  aria-labelledby="password-info"
+                />
+              </div>
+              <ToolTipModalContainer
+                modalIsOpen={passwordModalIsOpen}
+                closeModal={closeModal}
+                label="show-password-info-dialog"
+                content="Salasanassa tulee olla vähintään 10 merkkiä, ja siinä pitää
+                olla isoja kirjaimia, pieniä kirjaimia ja numeroita."
+              />
+            </div>
+            <div className="error-text">
+              {errors.password &&
+                errors.password.type === 'required' &&
+                'Kirjoita salasana'}
+              {errors.password &&
+                errors.password.type === 'pattern' &&
+                'Salasanan on oltava vähintään 10 merkkiä pitkä ja siinä pitää olla isoja kirjaimia, pieniä kirjaimia ja numeroita.'}
+              {errors.password &&
+                errors.password.type === 'maxLength' &&
+                'Salasanan on oltava enintään 30 merkkiä pitkä.'}
+            </div>
+          </div>
+
+          <div className="formfield-container">
+            <ValidatedInputField
+              label="Salasana uudestaan"
+              name="passwordConfirm"
+              ref={register({
+                required: true,
+                validate: value => {
+                  return value === watch('password')
+                },
+              })}
+              type="password"
+              ariaInvalid={!!errors.passwordConfirm}
+              inputClassName="create-account-input-text"
+              labelClassName={
+                errors.passwordConfirm
+                  ? 'create-account-errors-field'
+                  : 'create-account-input-field'
+              }
+            />
+            <div className="error-text">
+              {errors.passwordConfirm &&
+                errors.passwordConfirm.type === 'required' &&
+                'Kirjoita salasana uudestaan'}
+              {errors.passwordConfirm &&
+                errors.passwordConfirm.type === 'validate' &&
+                'Salasanat eivät ole samat'}
+            </div>
+          </div>
+          <ServiceRulesContainer setRulesAccepted={setRulesAccepted} />
+          <button type="submit" className="create-account-button">
```

Comment:
Hmm, I added those in, but the button is still not highlighted when using tab key to enter the form. Everything works though. I noticed that the login page has the same problem (kirjaudu button not getting highlighted) even though it uses the ButtonContainer component. From a quick google it seems that it might be a browser/MacOS issue. Here I didn't use the ButtonContainer-component since I wanted type to be submit for the form, and the ButtonContainer component automatically sets the type to button.

Reply:
It's possible to override highlight and outline styles when element is focused with css. For some reason there is `outline: none` at least in "Kirjaudu" button, which disables the outline focus. I have no idea why we added those overrides in the first place :D This might be the case with other ones too.